react/Viewer/Readme.md

```diff
@@ -6,10 +6,11 @@ The `Viewer` can display an **information panel** to show additional information
 
```

Comment:
You should add in the Readme that: 
- To have a working footer, the app need to have a CozySharing Provider. If the footer will be used on MobileApp, the app should have this Cordova plugin https://github.com/cozy/cozy-ui/pull/1914/commits/4db7f8fba866bffe04d81d42c716a8dea5c50157#diff-8c7901258747b81ed60cc2d9cbb254344fae11f8a602e56c1ae42d9eef11d318R50 
- To have the panels, the app need to have Harvest installed